.github/workflows/changeset.yaml

```diff
@@ -29,27 +29,23 @@ jobs:
       - name: Push git tags
         if: steps.changesets.outputs.published == 'true'
         run: git push --follow-tags
-  slack-notify:
+  notify:
     runs-on: ubuntu-latest
     needs: version
     steps:
-      - name: Notify Slack (success)
+      - name: Publish successful
         if: needs.version.outputs.published == 'true'
-        # https://github.com/marketplace/actions/post-to-slack
-        uses: pcolby/post-to-slack@v1
+        uses: slackapi/slack-github-action@v1.24.0
         with:
-          url: ${{ secrets.SLACK_WEBHOOK_URL }}
-          channel: "#wol_kaizen"
-          username: "Release"
-          text: Complete
-          iconEmoji: ":squid:"
-      - name: Notify Slack (failure)
+          channel-id: "C02NUQ27G56"
+          slack-message: "Packages have been published"
```

Comment:
Any possibility of a ternary for this message? Or a pre-step which sets it as a variable?

Reply:
Oh I'm not sure there's conditional logic in here, just strings. Their examples are more geared to updating an existing message with the status rather than adding conditional logic.